### ISSUE TITLE
Fail to build with go 1.14

### ISSUE THREAD

**Issue by Gisson** [Unbabel/replicant#25]
**Describe the bug**
The project fails to build with go 1.14, outputting the following error:
```
driver/go/driver.go:28:2: build constraints exclude all Go files in /src/replicant/vendor/github.com/containous/yaegi/stdlib
make: *** [Makefile:8: build] Error 1
```

**To Reproduce**
The easiest and most efficient way is to change the Dockerfile version from 1.13 to 1.14. Afterwards run `docker build .`

**Expected behavior**
Normal build operation

**Desktop (please complete the following information):**
 - OS: Gentoo
 - GO version: `go1.14 linux/amd64`
 - GO env:
```
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/jorge/.cache/go-build"
GOENV="/home/jorge/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/jorge/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go/pkg/tool/linux_amd64"
GCCGO="/usr/bin/gccgo"
AR="ar"
CC="x86_64-pc-linux-gnu-gcc"
CXX="x86_64-pc-linux-gnu-g++"
CGO_ENABLED="1"
GOMOD="/home/jorge/github/replicant/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build160057016=/tmp/go-build -gno-record-gcc-switches"
```

**Comment by brunotm** [comment]
@Gisson added a few comments to fix deps on your pr. Thanks!

**Comment by brunotm** [comment]
fixed on #29, closing
